src/App.js

```diff
@@ -1,29 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
+import './App.scss';
```

Comment:
- [ ] Kindly remove the `import './App.scss';` as there is no file with this name and this is causing an error.
- [ ] Kindly remove the `import './index.css';` line in the `index.js` file as there is no file with this name and this is causing an error.

Reply:
Noted and updated